admin/src/main/scala/io/buoyant/admin/names/DelegateApiHandler.scala

```diff
@@ -158,29 +159,29 @@ object DelegateApiHandler {
     def writeBuf[T](t: T): Buf = Buf.ByteArray.Owned(mapper.writeValueAsBytes(t))
   }
 
+  def getDelegateRsp(dtab: String, path: String, interpreter: NameInterpreter): Future[Response] =
```

Comment:
at some point we had separate delegator/{core,http} projects?  seems like linkerd-admin and namerd-iface-http-controller should pull in delegator-http?